payments-ui-core/src/main/assets/lpms.json

```diff
@@ -134,18 +134,18 @@
     "async": true,
     "fields": [
       {
-        "type": "billing_address",
-        "valid_country_codes": [
+        "type": "country",
```

Comment:
Rather than specify billing address, and display_fields, we will use a CountrySpec.  Sofort is a little weird in that it requires the country only in sofort[country] and billing_details[address][country] is not needed.